styles/cart.css

```diff
@@ -0,0 +1,101 @@
+.search_navbar {
+  padding: 10px;
+  width: var(--width-100);
+}
+.cake-logo {
+  color: var(--color-white);
+  font-size: var(--size-7);
+}
+.fa-search {
+  position: absolute;
+  top: 43%;
+  left: 26%;
+  cursor: pointer;
+}
+.list {
+  justify-content: space-around;
+}
+.list_end {
+  /* padding-left: 10%; */
+}
+.footer {
+  display: flex;
+  bottom: 0;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: var(--width-100);
+  height: 12vh;
+  background-color: var(--color-navyBlue);
+  gap: var(--size-2);
+}
+.footer-list {
+  display: flex;
+  gap: var(--size-3);
+  color: var(--color-white);
+}
+.footer-list_item {
+  list-style-type: none;
+  color: var(--color-white);
+  font-size: var(--size-6);
+}
+.footer-list_item--link {
+  color: var(--color-white);
+}
+.footer-h3 {
+  color: var(--color-white);
+}
+.heading-title {
+  color: var(--color-white);
+  font-size: var(--size-5);
+}
+.logo {
+  display: flex;
+  gap: var(--size-2);
+}
+.cart-container {
+  display: flex;
+  gap: var(--size-2);
+  width: 100%;
+  overflow: visible;
+  align-items: center;
+  justify-content: center;
+}
+
+.main-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 10%;
```

Comment:
you can remove this `margin-top: 10%; `as you're using `margin: 10% 0 10% 0;` on next line

Reply:
sure will do that